internal/databroker/server.go

```diff
@@ -122,7 +133,11 @@ func (srv *Server) Get(ctx context.Context, req *databroker.GetRequest) (*databr
 		Str("id", req.GetId()).
 		Msg("get")
 
-	record := srv.getDB(req.GetType()).Get(ctx, req.GetId())
+	db, err := srv.getDB(req.GetType())
+	if err != nil {
+		return nil, err
+	}
+	record := db.Get(ctx, req.GetId())
```

Comment:
can't all these methods return errors too?

Reply:
@calebdoxsey Yes, but current backend interface does not return error, let do it in other PR